### PR TITLE
Add download

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/annotation-cache/downloadsnpeffcache/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,7 +16,7 @@ jobs:
   test:
     name: Run pipeline with test data
     # Only run on push if this is the nf-core dev branch (merged PRs)
-    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'annotation-cache/downloadsnpeffcache') }}"
+    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'annotation-cache/downloadvepcache') }}"
```
Comment (maxulysse) [inline .github/workflows/ci.yml]:
```suggestion
    if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'annotation-cache/downloadsnpeffcache') }}"
```

File: nextflow.config [hunk 2]
```diff
@@ -184,12 +172,9 @@ plugins {
...
 
-// Load igenomes.config if required
-if (!params.igenomes_ignore) {
-    includeConfig 'conf/igenomes.config'
-} else {
-    params.genomes = [:]
-}
+// Load DSL2 modules config
+includeConfig 'modules/nf-core/snpeff/download/nextflow.config'
+
```
Comment (maxulysse) [inline nextflow.config]:
```suggestion
```

File: nextflow.config [hunk 3]
```diff
@@ -233,9 +218,6 @@ manifest {
     doi             = ''
 }
```
Comment (maxulysse) [inline nextflow.config:217]:
```suggestion

// Load DSL2 modules config
includeConfig 'modules/nf-core/snpeff/download/nextflow.config'

```

**Review by jfy133 — APPROVED** [review]
Same as VEP

**Pull request merged**